lib/bridge/MatrixHandler.js

```diff
@@ -674,6 +678,20 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
             return;
         }
     }
+    else if (cmd === "!bridgeversion") {
+        try {
```

Comment:
I'd rather we did the try catch around just ``const BridgeVersion = require("../../package.json").version; `` and set BridgeVersion to "Unknown" if the version could not be found.

```
let bridgeVersion = "Unknown";
try {
   bridgeVersion = require("../../package.json").version;
}
catch (err) {
    req.log.error("Failed to get bridge version", err.stack);
}
let notice = new MatrixAction(
    "notice",
    `Bridge Version: ${bridgeVersion}`
);
yield this.ircBridge.sendMatrixAction(adminRoom, botUser, notice, req);